pnpm-lock.yaml

```diff
@@ -43,7 +43,7 @@ importers:
     dependencies:
       '@typescript-eslint/utils':
         specifier: ^5.61.0
-        version: 5.62.0(eslint@8.57.1)(typescript@5.3.3)
+        version: 5.62.0(eslint@8.57.1)(typescript@5.0.4)
```

Comment:
The typescript version was downgraded after applying `pnpm add` for EDR's version bump